sdks/java/container/license_scripts/license_script.sh

```diff
@@ -16,6 +16,10 @@
 
 set -e
 
+echo "Hello, license_script.sh. Returning 0 for troubleshooting"
+exit 0
```

Comment:
It worked:

```
22:57:03 > Task :sdks:java:container:pullLicenses
22:57:03 Hello, license_script.sh. Returning 0 for troubleshooting
22:57:03 
22:57:03 > Task :sdks:java:container:java8:copyJavaThirdPartyLicenses NO-SOURCE
```

https://ci-beam.apache.org/job/beam_PostCommit_Java_PVR_Spark_Batch_PR/244/console